@kiva/kv-components/vue/KvCarousel.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+	<section
+		ref="KvCarousel"
+		class="tw-overflow-hidden tw-w-full"
+		aria-label="carousel"
+	>
+		<!-- Carousel Content -->
+		<div
+			class="tw-flex"
+			@click.capture="onCarouselContainerClick"
+		>
+			<kv-carousel-slide
+				v-for="index in numberOfSlides"
+				:key="index"
+				:aria-label="`slide ${index} of ${numberOfSlides}`"
+				:aria-current="true ? currentIndex + 1 === index : false"
+				:aria-hidden="ariaHidden(index)"
+				:tab-index="ariaHidden(index) ? '-1' : false"
+				:class="multipleSlidesVisible ? 'tw-mx-2 last:tw-mr-0 first:tw-ml-0' : 'tw-w-full'"
```

Comment:
ooh, haven't seen the `last:`, `first:` tailwind syntax before.

Not sure it would work with the multipleSlidesVisible part, but you might be able to do `tw-gap-x-2` on the tw-flex node above and forget about the first child last child stuff

Reply:
`last:` + `first:` may need to be "activated" via tailwind.config
https://tailwindcss.com/docs/hover-focus-and-other-states#last-child

Reply:
> `last:` + `first:` _would_ need to be "activated" via tailwind.config
> https://tailwindcss.com/docs/hover-focus-and-other-states#last-child

This is not the case, it seems to be working fine. I will modify for @ryan-ludwig suggestion on using the gap, though, its a little more semantic

Reply:
Yeah, @mcstover in JIT all variants are enabled. Since it's still in beta/experimental the main docs don't call out any JIT caveats, which can be troublesome. They list them as a whole here: https://tailwindcss.com/docs/just-in-time-mode

Hopefully this will become the stable version fairly soon and the docs will be updated on the rest of the site.

Reply:
Ok cool, thanks!